test/bundle.test.ts

```diff
@@ -19,7 +19,7 @@ describe.skipIf(process.env.SKIP_BUNDLE_SIZE === 'true' || process.env.ECOSYSTEM
   for (const outputDir of ['.output', '.output-inline']) {
     it('default client bundle size', async () => {
       const clientStats = await analyzeSizes('**/*.js', join(rootDir, outputDir, 'public'))
-      expect.soft(roundToKilobytes(clientStats.totalBytes)).toMatchInlineSnapshot(`"103k"`)
+      expect.soft(roundToKilobytes(clientStats.totalBytes)).toMatchInlineSnapshot(`"145k"`)
```

Comment:
This is buffer overhead :( Probably testing with something smaller...

Reply:
That's unfortunate! But then we can keep it behind a flag I suppose? Or would you consider it too unreliable/unsafe?

Reply:
For now i have added an example how to polyfill it is rather simple for end-users and also makes it easier for Nuxt side to iterate and eventually do it by default.

Besides Vue to get rid of buffer, and us to find smarter injections (I'm not sure easily possible), I also opened https://github.com/unjs/unenv/issues/160 to reduce Buffer polyfill size more but still even 1Kb is unacceptable for users don't need Buffer.